modules/porous_flow/include/dirackernels/PorousFlowPointEnthalpySourceFromPostprocessor.h

```diff
@@ -0,0 +1,44 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "DiracKernel.h"
+
+class SinglePhaseFluidProperties;
+
+/**
+ * Point source that adds fluid at a mass flux rate (specified by a postprocessor) and given
+ * temperature (specified by a postprocessor).
```

Comment:
Point source that adds heat energy corresponding to adding a fluid with flux rate specified by a postprocessor at given temperature (specified by a postprocessor).